pages/dev/governance/community-pool-proposals.mdx

```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
 3. Once the feedback window has passed, you can submit the Proposal for an on-chain governance vote.
 
 ## On-chain governance proposal
-Once a Governance Forum proposal has been live for the required temperature check and the feedback window has closed, you can submit an on-chain proposal to be voted on. Governance proposals are submitted on-chain using `axelard` (the [Axelar CLI](/learn/cli)) and a local Axelar node.
+Once a Governance Forum proposal has been live for the required temperature check and the feedback window has closed, you can submit an on-chain proposal to be voted on. Governance proposals are submitted on-chain using `axelard` (the [Axelar CLI](/learn/cli)).
 
 ### Prerequisites
 
 * You must have [axelard](/node/config-node#download-binaries-and-initialize-configuration)
-* You must have a local Axelar node running
-* You must have an account named `validator` with at least **2000AXL** balance
+* You must have an account with at least **2,000AXL** balance
```

Comment:
or find a community member that can fund the 2000 AXL deposit on your behalf within 1 hour of posting a proposal.